ayon_api/entity_hub.py

```diff
@@ -8,6 +8,10 @@
 from .utils import create_entity_id, convert_entity_id, slugify_string
 
 
+class InvalidValueError(ValueError):
```

Comment:
InvalidValueError kind of sounds like what `ValueError` is basically for. Should this be `NonExistentValueError`, `NonExistingValueError` or something? I wonder if this should be more descriptive?